### DISCUSSION TITLE
Proposal: let's crowdfund the development of a better "Link map" plugin (notes as a nodes in a graph)

### DISCUSSION THREAD

**Discussion by quesada** [zim-desktop-wiki/zim-desktop-wiki#1901] [Ideas]
I'm willing to put in say 100 eur to get to a better plugin that shows notes as a nodes in a graph.

"Before the 2020s, a note was just a note. A note was something you took, used, and tossed.

Let's call this: "Churn and Burn".

But now, in the Age of the [[Linked]] Note things are different. Now notes can grow and evolve with other notes, to form
a living system of related thoughts that you can develop throughout your lifetime.

Let's call this: "Know and Grow".
https://www.linkingyourthinking.com/

I find that very interesting. Yeah, notetakers have taken a leap forward and we are only starting to understand how Personal Knowledge Management (PKM)) works.
Funny that there are people selling products in the 1000s of $ to teach you how to organize your notes. Not endorsing the course, I haven't even finished reading the landing page :)

What's clear is that liking notes is super valuable. Zim had this far earlier than most. It's a wiki, after all. Where the market is moving is visualizing notes and connections, and there, Zim has arguably the least feature-ladden solution: It's only static snapshots with xdot.

Who here would be willing to chip in to pay a developer (anyone volunteering?) to write such plugin? @jaap-karssenberg  has too much on his plate and has already said that this feature is not high in his priority list. Anyone willing to pay to build it? We can then brainstorm features etc

**Comment by introt** [comment]
Feel free post to link a bounty site of your choosing! There's one open for markdown support on [bountysource](https://app.bountysource.com/issues/45030698-support-markdown-and-display-files-with-md-extension-in-the-index); those backers might be willing to chip in.

**Reply by quesada** [reply]
Will do, thanks for the hint!

**Reply by quesada** [reply]
About markdown support: I'm using now @pfalcon  fork with it! it seems to be usable, he uses it for day to day duty. There are some gitches but it fees mature enough that it could be a great candidate to be included into the main repo (@jaap-karssenberg , this is of course your decision; I'd be happy to do some exhaustive testing).

This 'solves' the visualization problem because one can use obsidian, logseq or any other tool with a more developed 'link map' to do network operations, while keeping the fantastic fast, native, bug free experience that zim is to actually write. Which is 90% of the time :)

**Comment by quesada** [comment]
FWIW, I moved away from zim to Amplenote (which also doesn't have a workable graph lol)

**Reply by introt** [reply]
Glad you've found a suitable app! Hopefully they'll get their graphs worked out sooner than later :)
